packages/web-app/src/controllers/review-appeal-submission.js

```diff
@@ -30,7 +31,15 @@ const getReviewAppealSubmission = (req, res) => {
 
 const postReviewAppealSubmission = (req, res) => {
   const reviewOutcome = req.body['review-outcome'];
-  const nextPage = reviewOutcome === 'valid' ? validAppealDetails : home;
+
+  let nextPage;
+  if (reviewOutcome === 'valid') {
+    nextPage = validAppealDetails;
+  } else if (reviewOutcome === 'invalid') {
+    nextPage = `${invalidAppealDetails}/${req.session?.appeal?.appeal?.id}`;
+  } else {
+    nextPage = home;
```

Comment:
Can we please add a unit test to cover the else clause?

This would be the case where `reviewOutcome = 'incomplete'`